src/main/resources/signs.yml

```diff
@@ -53,3 +53,14 @@ leave:
 - '&a<arena-name>'
 - '---------------'
 - 'Hit to leave'
+
+ready:
+- 'Ready:'
+- '<ready-1>'
+- '<ready-2>'
+- '<ready-3>'
+notready:
```

Comment:
Nice touch with some predefined templates!

However, since these two templates are effectively decoupled rather than being parts of the same template (like with the `join` and `join-running` example), there should be an extra linebreak between them so they aren't "grouped". I hope that makes sense.